infra/analytics/service/sfn_sprint_reports.tf

```diff
@@ -0,0 +1,106 @@
+# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_log_group
+resource "aws_cloudwatch_log_group" "sprint_reports" {
+  name_prefix = "/aws/vendedlogs/states/${local.service_name}-sprint-reports"
+
+  # Conservatively retain logs for 5 years.
+  # Looser requirements may allow shorter retention periods
+  retention_in_days = 1827
+
+  # checkov:skip=CKV_AWS_158:skip requirement to encrypt with customer managed KMS key
+}
```

Comment:
We might want to backport this logs group pattern to the other step function. It fails to deploy periodically without this.

Reply:
Sounds like a good idea! Is that something you want to do as part of this PR, or another one?

Reply:
https://github.com/HHS/simpler-grants-gov/pull/1615

Reply:
^ New PR for the oracle SFN logs